crates/solvers/src/run.rs

```diff
@@ -47,28 +48,32 @@ async fn run_with(args: cli::Args, bind: Option<oneshot::Sender<SocketAddr>>) {
                 dex::Dex::ZeroEx(
                     dex::zeroex::ZeroEx::new(config.zeroex).expect("invalid 0x configuration"),
                 ),
-                config.base,
+                config.base.clone(),
+                RateLimiter::new(config.base.rate_limiting_strategy, "dex_api".to_string()),
```

Comment:
Why calling `RateLimiter::new` here, if we can do it inside `solver::Dex::new`? (it depends on the config only)